src/SFA.DAS.EmployerCommitmentsV2.Web/DependencyResolution/IoC.cs

```diff
@@ -8,8 +10,11 @@ public static class IoC
     {
         public static void Initialize(Registry registry)
         {
+            registry.IncludeRegistry<AutoConfigurationRegistry>();
```

Comment:
Does `AutoConfigurationRegistry` need to be included here?

Reply:
removed